policies/gac/networks.py

```diff
@@ -376,9 +379,7 @@ def train(self, transitions, action_sampler, actor, critic, K):
         Line 14 of Algorithm 2.
         Get the Q value of the states and action samples.
         """
-        Q1, Q2 = critic(
-            tf.concat([states, actions], -1)
-        )
+        Q1, Q2 = critic(states, actions)
```

Comment:
This line is pretty dangerous. I mean, it assumes that the critic returns values, which will only happen when num_critic = 2. So why we give the option num_critic = 1 then?

Reply:
This is the wired stuff from the other implementation I guess. We can definitely change it.